bentoctl_azfunctions/create_deployable.py

```diff
@@ -98,6 +106,7 @@ def create_deployable(
                     i is not None
                     for i in bentoml_cattr.unstructure(info.conda).values()
                 ),
+                enable_grpc=enable_grpc,
```

Comment:
same as the lambda PR. make this a config. And confirm if the platform support gRPC